index.html

```diff
@@ -14,14 +14,20 @@
     <head>
         <nav>
             <a src="" id="logo">My Logo</a>
+            <ul class="menu-items">
```

Comment:
[OPTIONAL]
- [ ] Kindly note that the link tag `a` does not have a `src` attribute, I believe you meant to use `href` instead.

Reply:
My bad will fix it